File: PRAK205-2510817110012-Muhammad Dani Maulana.c
#include <stdio.h>
#include <math.h>

int main(){
    int tinggi, miring, alas, K, L;
    printf("Masukkan Sisi A: ");
    scanf("%d", &tinggi);
    printf("Masukkan Sisi B: ");
    scanf("%d", &miring);

    alas = sqrt(pow(miring,2) - pow(tinggi,2));
    K = alas + tinggi + miring;
    L = alas * tinggi / 2;
    printf("Alas: %d cm\n",alas);
    printf("Tinggi: %d cm\n",tinggi);
    printf("Keliling: %d cm\n",K);
    printf("Luas: %d cm^2",L);
    return 0;
}

File: PRAK205-2510817110012-Muhammad Dani Maulana.py
import math
Tinggi = int(input("Masukkan Sisi A: "))
Miring = int(input("Masukkan Sisi B: "))
Alas = math.sqrt(pow(Miring,2) - pow(Tinggi,2))
Keliling = Alas + Miring + Tinggi
Luas = Alas * Tinggi / 2
print("Alas: %d cm"%Alas)
print("Tinggi: %d cm"%Tinggi)
print("Keliling: %d cm"%Keliling)
print("Luas: %d cm^2"%Luas)


